mantidproject/mantid
# Engg GUI: show some customised plots when the calibration finishes

Fixes #13377

After calibration has been carried out there should be few `Vanadium Curves` and `Ceria` peaks plotted

**To Test**
When `calibrate` button has been clicked, let the process complete and open completion check:
- All the spectrum have been plotted for `engggui_vanadium_curves_ws`
- `Engg Gui Difc Zero Peaks Bank 1` and `Engg Gui Difc Zero Peaks Bank 2` 
  - workspaces produced
  - plots graph with: 
    - appropriate labels
    - each graph should produces a dotted line and straight line
    - Calibration check-box disabled during process and remembers settings
- Cropped calibration produces correct ceria plots for the banks selected
  - ceria plots when `Spectrum-IDs` are entered
  - with single bank 
  - appropriate title 
- Window replaces the `vanadium curves`
- Deletes and plots the `Engg GUI Difc Zero` & `fitting peaks`
- Documentation and EnggDiffractionPresenterTest Unit Test has also been updated
- Issue fixed where cropped calib bank was over-writing bank 1
  - Now saves SpectrumIDs bank name as `engggui_calibration_bank_cropped` 

Release Notes: http://www.mantidproject.org/ReleaseNotes_3_7_Diffraction#Engineering_Diffraction
